packages/service-library/src/servicelib/async_utils.py

```diff
@@ -12,6 +13,9 @@ class Context:
     initialized: bool
 
 
+contexts = {}
```

Comment:
I'd rename this to something a bit more meaningful

Reply:
ok done